src/toICN-core.js

```diff
@@ -45,7 +52,7 @@ module.exports = function(raw){
         unSupported = true;
       }
     }
-    s = no+(swapped?"~":"")+(isQAvailable?("["+q+"]"):""+(unSupported?"[!!"+q+"!!]":""));
+    s = no+(swapped?"~":"")+(isQAvailable?("["+q+"]"):""+(unSupported?"[!!"+q+"!!]":"")+(isOnChord?"/"+onChordNo:""));
```

Comment:
isONChordはあまり役に立っていないようなので 
```suggestion
    s = no+(swapped?"~":"")+(isQAvailable?("["+q+"]"):""+(unSupported?"[!!"+q+"!!]":"")+(onChordNo!=""?"/"+onChordNo:""));
```

にして isOnChordの処理を消しませんか？